### ISSUE TITLE
plugin does not work when --kubeconfig provided as command line paramter

### ISSUE THREAD

**Issue by FingerLiu** [kubernetes/kubectl#1032]
**What happened**:
When  `--kubeconfig` is provided as a command line  parameter, kubectl does not recognize plugin any more.

**What you expected to happen**:
Plugin should work when  `--kubeconfig` is provided.

**How to reproduce it (as minimally and precisely as possible)**:

1 make a minimal kubectl plugin
```bash
cat <<EOF >> /usr/local/bin/kubectl-ls
#!/bin/bash
ls
EOF
chmod +x /usr/local/bin/kubectl-ls
```

2 this works.
```bash
kubectl ls
```

3 and this do not.
```bash
kubectl --kubeconfig=~/.kube/config ls
# Error: unknown command "ls" for "kubectl"
```

4 but if you append kubeconfig to the end, it works well.
```bash
kubectl ls --kubeconfig=~/.kube/config
```

**Environment**:
- Kubernetes client and server versions (use `kubectl version`):Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T18:49:28Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.8", GitCommit:"ec6eb119b81be488b030e849b9e64fda4caaf33c", GitTreeState:"clean", BuildDate:"2020-03-12T20:52:22Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):mac/linux

Is this by desgin or bug? If this should be fixed, I will submit the PR.

**Comment by dty1er** [comment]
It seems like this is already fixed. https://github.com/kubernetes/kubectl/issues/884
The change is included in the latest kubectl.

**Comment by FingerLiu** [comment]
close due to fixed in 1.20 
https://github.com/kubernetes/kubernetes/pull/92343
